### ISSUE TITLE
Possible typo in Chapter 3

### ISSUE THREAD

**Issue by mccarthy-m-g** [keithmcnulty/ona_book#5]
Hi Keith,

I think, but am not sure, there is a typo in Chapter 3, in relation to Figure 3.1. The picture below is the text after that figure. 

The edge pairs don’t seem to line up with what’s in the graph. In both equations Suraya and Jane are paired twice, and David is never paired with Jane even though they share an edge in the graph.

![7FCEDB33-464F-4CD8-B6C0-D8F44E6828F4](https://user-images.githubusercontent.com/51542091/137393364-516c3bed-380c-47e6-b313-bab548da3aa7.png)

**Comment by keithmcnulty** [comment]
You’re right.  It should be David <—> Jane and not Suraya <—> Jane.  Thanks for spotting this, must have slipped through from an earlier draft.  I’ll fix it right away.
